stix2/v20/observables.py

```diff
@@ -706,6 +706,19 @@ class WindowsRegistryValueType(_STIXBase):
     ])
 
 
+class CallableValues(list):
```

Comment:
@clenk, given that this class is used by both `v20` and `v21` objects, can it be placed in a common location to prevent duplicate code?